### ISSUE TITLE
Need to first face detection?

### ISSUE THREAD

**Issue by hanoi2018** [MarekKowalski/DeepAlignmentNetwork#32]
Hi MarekKowalski,
Sorry for this question, but I am a newbie in this field, I wonder what if need to firstly face detection?
Thanks!

**Comment by MarekKowalski** [comment]
Hi,

I am having trouble understanding your question, could you rephrase it?

Thanks,

Marek

**Comment by hanoi2018** [comment]
Hi,
I am sorry, It means if I use DAN for the face alignment problem so I need to execute face detection? or the DAN algorithm has face detection on it?

Thanks!

**Comment by hanoi2018** [comment]
Hi Marek,
Thank you for your response, because OpenCV's face detection not exactly for some situation so now I    will use deep learning for face detection problem, but I am considering I will have more capability in my app :( Do you have any suggestion for me?
Thanks!

**Comment by MarekKowalski** [comment]
Hi,

I am not sure what you mean by " I am considering I will have more capability in my app ", could you elaborate?

**Comment by hanoi2018** [comment]
Hi Marek,
That means when I use this model for my app (on mobile) which use more Ram

**Comment by MarekKowalski** [comment]
Hi,

You might want to consider a method that does both face detection and alignment, such as this:
https://arxiv.org/abs/1604.02878

Marek

**Comment by hanoi2018** [comment]
HI Marek, 
Thanks for your response, I will try it 👍
